src/ReactWrapper.jsx

```diff
@@ -224,13 +225,14 @@ export default class ReactWrapper {
    * NOTE: can only be called on a wrapper instance that is also the root instance.
    *
    * @param {Object} state to merge
+   * @param {Function} cb - callback function
    * @returns {ReactWrapper}
    */
-  setState(state) {
+  setState(state, cb) {
```

Comment:
since this param is optional, it should default to `undefined` so it doesn't count towards the function's length (same throughout)